### DISCUSSION TITLE
Using pg-promise in a project with pg

### DISCUSSION THREAD

**Discussion by cesco69** [vitaly-t/pg-promise#936] [General]
I want use  "pg-promise" but my project has already "pg". I can set up "pg-promise" to use same pool of  "pg" without establishing a new connection?

**Comment by vitaly-t** [comment]
`pg-promise` encapsulates the `pg` driver, so it can only work the other way round. Remove `pg` dependency from your project, add `pg-promise` only, you then can access `pg` for that compatibility, via `pgp.pg` property.
